### ISSUE TITLE
Currently building jobs are not displayed in JobBuilds widget

### ISSUE THREAD

**Issue by macklinu** [plouc/mozaik-ext-jenkins#14]
| item | info | notes |
| --- | --- | --- |
| **node** version | v5.9.1 | _output from `node --version`_ |
| **npm** version | 3.7.2 | _output from `npm --version`_ |
| **mozaik-ext-jenkins** version | 1.655 | _available in project's `package.json`_ |
| **jenkins** version | 1.655 | _your jenkins server version_ |
| **mozaik** version | ^1.1.0 | _available in project's `package.json`_ |
| **mozaik-demo** version | N/A | _version of the demo used, depends on which method you used to setup your Mozaïk dashboard_ |
| **component** | jenkins.job_builds | _name of the extension's component or `client` if it's related to the extension's client_ |
| **browser** | Chrome 49.0.2623.87 | _browser used, applyable if the issue is not related to the client_ |
## Expected behavior

While a job is building, display the job in the list of job builds with the status **building**.
## Actual behavior

The building job is not displayed in the list.
## Steps to reproduce

(This is somewhere between a bug report and a feature request and would like your feedback.)

Trigger a Jenkins job and refresh your dashboard. This requires a little bit of timing, depending on how long your Jenkins job runs. I've found in certain conditions that visiting the dashboard while a certain job is building prevents any `JobBuild` components from being displayed in the `JobBuilds` widget.

In this case, I see:

<img width="423" alt="2016-03-29 at 3 10 pm" src="https://cloud.githubusercontent.com/assets/2344137/14120421/c21f349c-f5c0-11e5-99ac-5801a53fcdb4.png">

When no jobs are building, I see all job builds in the list, as expected.

<img width="424" alt="2016-03-29 at 3 10 pm success" src="https://cloud.githubusercontent.com/assets/2344137/14120432/d14bd7f4-f5c0-11e5-8a7d-6b9b15d767c1.png">

I noticed that the `JobBuild` component takes the `build.result` and displays it with `toLowerCase()` ([code is here(https://github.com/plouc/mozaik-ext-jenkins/blob/v1.1.0/src/components/JobBuild.jsx#L9))

According to the Jenkins API endpoints I am hitting, which I believe to be the case for all versions of the Jenkins API:

**Building job object**

``` json
{
  "building" : true,
  "duration" : 0,
  "id" : "1",
  "number" : 1,
  "result" : null,
  "timestamp" : 1459278308034,
  "builtOn" : ""
}
```

**Completed job object**

``` json
{
  "building" : false,
  "duration" : 10025,
  "id" : "1",
  "number" : 1,
  "result" : "SUCCESS",
  "timestamp" : 1459278308034,
  "builtOn" : ""
}
```

For a building job, `result` == null, where I think the weird behavior is happening where no jobs are shown when `build.result.toLowerCase()` is called in `JobBuild`. I don't see any logs for this, though, so this is a guess.

So long story short, I guess this is a feature request for properly displaying the pending build status of a `JobBuild`. I would be glad to look into this, as I would like this for my company's dashboard.

Let me know if you have any questions or input about this behavior/idea.
